eu.ad5001.LogarithmPlotter.json

```diff
@@ -0,0 +1,64 @@
+{
+    "app-id": "eu.ad5001.LogarithmPlotter",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "finish-args": [
+        "--share=ipc",
+        "--device=dri",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--talk-name=org.freedesktop.DBus.Proprieties",
```

Comment:
I think you mean properties https://dbus.freedesktop.org/doc/dbus-specification.html#standard-interfaces-properties

Reply:
Can't believe I missed that! Nice catch.